### ISSUE TITLE
TypeError: unsupported operand type(s) for +: 'int' and 'NoneType'

### ISSUE THREAD

**Issue by rufik** [StyraHem/hass#17]
I've just connected new Shelly 1 PM (firmware 20190423-080637/v1.4.9-switch1pm-hotfix4@f8c51629) which was discovered successfully. But energy (watt) sensor has "Unknown" value all the time. HA version is 0.92.1 and logs says:

```
2019-05-16 00:09:52 ERROR (Thread-2) [pyShelly] Error receiving UDP: unsupported operand type(s) for +: 'int' and 'NoneType', Traceback (most recent call last):
  File "/config/deps/lib/python3.7/site-packages/pyShelly/__init__.py", line 777, in _udp_reader
    self.blocks[id].update(data, addr[0])
  File "/config/deps/lib/python3.7/site-packages/pyShelly/__init__.py", line 91, in update
    dev.update(data)
  File "/config/deps/lib/python3.7/site-packages/pyShelly/__init__.py", line 379, in update
    watt = sum(data.get(pos) for pos in self._positions)
TypeError: unsupported operand type(s) for +: 'int' and 'NoneType'
Traceback (most recent call last):
  File "/config/deps/lib/python3.7/site-packages/pyShelly/__init__.py", line 777, in _udp_reader
    self.blocks[id].update(data, addr[0])
  File "/config/deps/lib/python3.7/site-packages/pyShelly/__init__.py", line 91, in update
    dev.update(data)
  File "/config/deps/lib/python3.7/site-packages/pyShelly/__init__.py", line 379, in update
    watt = sum(data.get(pos) for pos in self._positions)
TypeError: unsupported operand type(s) for +: 'int' and 'NoneType'
```

I'm using your component at **version: 0.0.12/0.0.29**

**Comment by hakana** [comment]
It is a bug in Shelly fw and will be supported in next firmware release.

**Comment by hakana** [comment]
Error fixed, will work fully in fw 1.5.X
